src/components/shared/Filter/ActivityFilter/index.jsx

```diff
@@ -104,36 +81,50 @@ const RuleFilter = ({
             />
           </Column>
           <Column className="filter-column" width={9} offset={4}>
-            {displayOptionInput(
-              filterBy,
-              option,
-              updateFilterOption,
-              isValidFilter,
-              handleOptionSelector
+            {isDropdown && (
+              <div className="filter-select">
+                <Select
+                  isRequired
+                  onChange={handleOptionSelector}
+                  placeholder="Select an Option"
+                  value={
+                    option && { label: option, value: option.toLowerCase() }
+                  }
+                  options={selectOptions(filterBy)}
+                />
+              </div>
+            )}
+            {isDatePicker && (
+              <div className="sg-date-picker">
+                <DateRangePicker
+                  small
+                  startDate={startDate}
+                  startDateId="startDate"
+                  endDate={endDate}
+                  endDateId="endDate"
+                  onDatesChange={onDateChange}
+                  focusedInput={focusedInput}
+                  onFocusChange={onFocusChange}
+                  isOutsideRange={() => false}
```

Comment:
What's this for since we're always defaulting to false on this function?

Reply:
This is so that the calendar allows you to click dates behind the current date. The default calendar blocks the user/greys out from choosing a date that has already passed.  I looked up on stackoverflow and people had solved it this way since I think usually that this function takes in a start/end date, and you return true/false depending if you want to show those dates.  @alfredlucero